pypy.py

```
@@ -0,0 +1,93 @@
+    sum = a + b
+    print(sum)  # Only prints the result, doesn't return it
+
+result = calculate_sum(5, 3)
+print(result)  # None, because the function doesn't return anything
+
+# Poor use of exception handling
+def read_file(file_path):
+    try:
+        file = open(file_path, 'r')  # No context manager
+        data = file.read()
+        return data
```

Ruff (0.14.0)

Consider moving this statement to an `else` block

(TRY300)